<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copter optimisation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_no_motors">
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_mass">
     <property name="toolTip">
      <string>Total mass of craft</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of motors</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="toolTip">
      <string>The minimum upwards acceleration that the system must be capable of.</string>
     </property>
     <property name="text">
      <string>Min upwards acceleration [m/s/s]</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Motors</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QListView" name="listView_props"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="listView_motors"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string>Total mass of craft</string>
     </property>
     <property name="text">
      <string>Craft weight [kg]</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QListView" name="listView_results"/>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_min_acc">
     <property name="toolTip">
      <string>The minimum upwards acceleration that the system must be capable of.</string>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="7">
    <widget class="QWidget" name="widget_plot" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="layout_plot"/>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Propellers</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>Nominal voltage of battery and system</string>
     </property>
     <property name="text">
      <string>Battery voltage [V]</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_voltage">
     <property name="toolTip">
      <string>Nominal voltage of battery and system</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="toolTip">
      <string>The percentage by which the combination is able to surpass the motor limit.</string>
     </property>
     <property name="text">
      <string>Max allowable current [%]</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_max_I">
     <property name="toolTip">
      <string>The percentage by which the combination is able to surpass the motor limit.</string>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>110.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_optimise">
     <property name="text">
      <string>Optimise and return top</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBox_number">
     <property name="toolTip">
      <string>Number of results to return</string>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
